snippets/sortMultiArrayByColumn.md

```diff
@@ -0,0 +1,28 @@
+---
+title:  sortMultiArrayByColumn
+tags: array,intermediate
+---
+
+Return sorted multiple array
+
+Use `array_multisort` to sort multiple array by column.
```

Comment:
This does not explain the snippet at all, it just describe a single line of code...